[PATCH] Fix build for recent toolchains

Starting with Linux 4.4 headers, mtd-user.h isn't including stdint.h
anymore which breaks the build.

Signed-off-by: Gary Bisson <gary.bisson@boundarydevices.com>
---
 src/BootControlBlocks.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/BootControlBlocks.h b/src/BootControlBlocks.h
index 192eb61..aacdb64 100644
--- a/src/BootControlBlocks.h
+++ b/src/BootControlBlocks.h
@@ -20,6 +20,8 @@
 #ifndef BOOTCONTROLBLOCKS_H_
 #define BOOTCONTROLBLOCKS_H_
 
+#include <stdint.h>
+
 #define NCB_FINGERPRINT1	0x504d5453	//!< 'STMP'
 #define NCB_FINGERPRINT2	0x2042434e	//!< 'NCB<space>' - NAND Control Block
 #define NCB_FINGERPRINT3	0x4e494252	//!< 'RBIN' - ROM Boot Image Block - N
-- 
2.6.4

